plugins/talk-plugin-notifications-category-staff/client/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I guess this file isn't required anymore? ;-)

Reply:
This one is, there's no way to reference the client eslint yet with patterns.

Reply:
True, my bad, got confused